bonan_richards_eqn/get_bonan_data.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This script downloads a specific version of a Matlab script from a GitHub repository
+# and generates data files for two different soil types using that script.
+# The output data files are saved in the paths bonanmodeling/sp_08_01/bonan_data_clay.txt
+# and bonanmodeling/sp_08_01/bonan_data_sand.txt, relative to the directory
+# where this script is run.
+
+# Parse the input argument for the output directory
+output_dir=$1
```

Comment:
If your script required command line arguments, it is good practice to add something like
```
[[ $# -lt 1 ]] && echo "Usage $0 <output_dir>" && exit 0
```
(schematically)
to inform the user when they don't provide all the arguments.